### DISCUSSION TITLE
3080 ti Nvidia driver recommendation

### DISCUSSION THREAD

**Discussion by fsitios** [NebuTech/NBMiner#688] [Q&A]
Hi, I'm using HiveOs and I need to know which driver version you recommend for 3080 ti cards. Also, what overclocking do you recommend? I am using 1150 Core, 2000 Mem and 285 Pil. Nvidia 470.74 Driver

Thanks,
Fernando

**Comment by Tom5051** [comment]
wouldn't that be a question for a HiveOS forum?
